### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by katranci** [jiminny/jiminny-waveform#9]
```
{
  "talkTimes": {
    "user": [
      [0, 2.5], [10, 12.5]
    ],
    "customer": [
      [5, 7.5]
    ]
  }
}
```

Given the test JSON above the expected outcome is the following:
![image](https://user-images.githubusercontent.com/61077/37603214-027e5b74-2b86-11e8-8131-0aa0de545e74.png)

Whereas I'm getting this instead:
![image](https://user-images.githubusercontent.com/61077/37603736-2b65e7f4-2b87-11e8-8b15-29186796c92d.png)

**Comment by notargetbg** [comment]
Yep.. Indeed it looks like a bug of some sort. I will try to find it. It seems I am missing subtracting something. Not sure if it is related to the way I use the `<WaveformActivePart>` to draw the waveform or my calculations. I will try to solve this one out.

**Comment by notargetbg** [comment]
Actually the mistake was in my `<Waveform>` component. It seems I forgot to use the total duration I calculate in the parent component and it was set separately for each channel. One length for the user channel and one for the customer. That is why it didn't work with the test json. Because I got 2 different total lengths, respectively - 12.5, and 7.5.
I've fixed it and now it outputs the same symmetrical visualization.
